GMG/Scenes/ChordProgress/ChordProgressView.swift

```diff
@@ -127,11 +147,35 @@ extension ChordProgressView {
                     .font(Typography.WantedSansStd.R4)
             }
             .foregroundStyle(
-                colorScheme == .light ? Color.black1 : Color.white1
+                editMode?.wrappedValue.isEditing == true
+                    ? Color.white1 : Color.black1
             )
         }
     }
 
+    struct EditController: View {
+        let canUndo: Bool
+        let canRedo: Bool
+        let onTapUndo: () -> Void
+        let onTapRedo: () -> Void
+        
+        var body: some View {
+            HStack(spacing: 24) {
+                Button {
+                    onTapUndo()
+                } label: {
+                    Image(canUndo ? "Undo" : "Undo-unactive")
+                }
+                
+                Button {
+                    onTapRedo()
+                } label: {
+                    Image(canRedo ? "Redo" : "Redo-unactive")
+                }
```

Comment:
Image에 `.renderingMode(.template)` 붙여주면 SF Symbol처럼 색상 변경이 가능합니다. 그래서 이미지는 하나만 에셋에 추가하고 색상 변경하는 식으로 사용하는 것이 좋을 것 같습니다!

그리고 String으로 초기화하는 것보다 Xcode에서 생성한 ImageResource Symbol(`ImageResource.undo`)로 초기화하면 좋을 것 같습니다.

```suggestion
                Button {
                    onTapUndo()
                } label: {
                    Image(.undo)
                        .renderingMode(.template)
                        .foregroundStyle(canUndo ? Color.white1 : Color.black1)
                }
                
                Button {
                    onTapRedo()
                } label: {
                    Image(.redo)
                        .renderingMode(.template)
                        .foregroundStyle(canUndo ? Color.white1 : Color.black1)
                }
```

Reply:
.renderingMode의 존재를 알려주셔서 감사합니다 🤯